runtime/simulation/Makefile

```diff
@@ -0,0 +1,91 @@
+ifeq ($(MAKECMDGOALS),)
+  $(shell pkill -f -u $(USER) questa)
```

Comment:
This requires a better guard, e.g. by storing the PID of the last simulation. Otherwise it will kill simulations independent of a TaPaSCo simulation.